cmd/av/stack_branch.go

```diff
@@ -159,6 +167,14 @@ func init() {
 	// See the comment on stackBranchFlags.Rename.
 	stackBranchCmd.Flags().
 		BoolVarP(&stackBranchFlags.Rename, "rename", "m", false, "rename the current branch")
+	stackBranchCmd.Flags().StringVar(
+		&stackBranchFlags.RetroactivelyRenameFrom,
+		"retroactively-rename-from",
```

Comment:
I think this flag name is too long/clunky.

Can we just add a command like `av branch-meta set old-branch --name new-branch`?

Reply:
Yeah. This is a bit intentional because I don't think this is something people use daily basis, but need it as a way to fix this. We hide branch-meta because its more like a debugging command, so I don't want it documented.